### PR TITLE
Update frame.max.html

### PR DESCRIPTION
**Pull request by karl1993**

### CODE REVIEW

**Comment by powerivq** [comment]
Hey @karl1993 . I don't quite understand the purpose of this PR. Can you explain?

File: 3p/frame.max.html [hunk 1]
```diff
@@ -1,10 +1,16 @@
 <!doctype html>
 <head>
+<script async custom-element="amp-auto-ads"
+        src="https://cdn.ampproject.org/v0/amp-auto-ads-0.1.js">
+</script>
 <meta charset="utf-8">
 <meta name="robots" content="noindex">
 <script src="./integration.js"></script>
 </head>
-<body style="margin:0">
```
Comment (karl1993) [inline 3p/frame.max.html:7]: >

**Review by karl1993 — COMMENTED** [review]
Fix it

**Review by powerivq — CHANGES_REQUESTED** [review]
I do not see why we need this here.

**Comment by powerivq** [comment]
Closing as inactive.

**Pull request closed without merging**